src/common/api/base.go

```diff
@@ -166,12 +106,12 @@ func (b *BaseAPI) Redirect(statusCode int, resouceID string) {
 func (b *BaseAPI) GetIDFromURL() int64 {
 	idStr := b.Ctx.Input.Param(":id")
 	if len(idStr) == 0 {
-		b.CustomAbort(http.StatusBadRequest, "invalid ID in URL")
+		b.CustomAbort(http.StatusBadRequest, utils.WrapErrorMessage(http.StatusBadRequest, "invalid ID in URL"))
```

Comment:
Suggest removing `b.CustomAbort` and refactor the return list of these utility functions.